src/main/java/xyz/aadev/generitech/common/network/messages/power/PacketSides.java

```diff
@@ -0,0 +1,68 @@
+package xyz.aadev.generitech.common.network.messages.power;
+
+import io.netty.buffer.ByteBuf;
+import net.minecraft.client.network.NetHandlerPlayClient;
+import net.minecraft.network.NetHandlerPlayServer;
+import xyz.aadev.aalib.common.network.PacketBaseThreadSafe;
+import xyz.aadev.generitech.GeneriTech;
+import xyz.aadev.generitech.common.tileentities.power.TileEntityPowerStorage;
+
+public class PacketSides extends PacketBaseThreadSafe {
+
+    private int side0;
+    private int side1;
+    private int side2;
+    private int side3;
+    private int side4;
+    private int side5;
+
+
+    public PacketSides() {
+
+    }
+
+    public PacketSides(int[] sides) {
+        this.side0 = sides[0];
+        this.side1 = sides[1];
+        this.side2 = sides[2];
+        this.side3 = sides[3];
+        this.side4 = sides[4];
+        this.side5 = sides[5];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1186)